data_preprocessing/collectivite_territoriale.py

```diff
@@ -100,25 +95,20 @@ def preprocess_colter_data(
     ] = "particulier"
 
     df_colter = pd.concat([df_colter, df_communes])
-
-    if os.path.exists(data_dir + "colter-new.csv"):
-        os.remove(data_dir + "colter-new.csv")
-
     df_colter.to_csv(data_dir + "colter-new.csv", index=False)
```

Comment:
maybe here you can save csv creating `data_dir` in the same time if not exist but I think we need to catch this possible problem

Reply:
I don't think I understand what you mean. I don't save the css anymore because everything is done in the initial DAG. The update DAGs are deactivated for now, until we are able to make sure they work every time. Is there a specific need to save the data in Minion other than the updates ?